index.html

```diff
@@ -1,36 +1,35 @@
- <!DOCTYPE html>
- <html lang="en">
- <head>
-     <meta charset="UTF-8">
-     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-     <title>Document</title>
-     <link rel="stylesheet" href="style.css">
```

Comment:
1) Add bootstrap library.
2) Todo UI should be vertically and horizontally, center aligned
3) Use bootstrap class name for input box and button.
4) Button background colour should be blue